### DISCUSSION TITLE
OpenVPN disconnects client after inactivity

### DISCUSSION THREAD

**Discussion by BJohnson98** [angristan/openvpn-install#1415] [Help]
So I have to clients, one windows 10 client, and a linux client. The windows client works perfectly, but the linux client gets disconnected after 20-40 minutes. The log on the clients side is "Failed TLS Handshake" 
I did not change the configuration files at all. I think it has to do with inactivity on the linux server client. is there any way I can change the config of the server or client so it does not re-verify clients or disconnects them for inactivity??

**Comment by randshell** [comment]
Try to add `keepalive 10 60` to `/etc/openvpn/server.conf`
